app.vue

```diff
@@ -38,9 +41,11 @@ const isDark = computed({
           </div>
           <a href="/">
             <div class="flex">
-              <UIcon
-                  name="i-heroicons-magnifying-glass-16-solid"
-                  class="text-primary w-10 h-10 hidden sm:block"/>
+              <img
+                  :src="logoPath"
+                  alt="logo"
+                  class="w-9 h-9 mx-auto"
+              />
```

Comment:
```suggestion
              <img :src="logoPath" alt="logo" class="w-9 h-9 mx-auto"/>
```

Keep this on one line.